packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -20,13 +20,14 @@ export const onRouteUpdate = ({ location }) => {
     window.ga(`send`, `pageview`)
   }
 
-  if (`requestAnimationFrame` in window) {
+  const delay = pluginOptions.pageTransitionDelay
```

Comment:
Let's rewrite to something like below. Let's move to setTimeout for all the things.

```suggestion
  const delay = Math.max(32, pluginOptions.pageTransitionDelay)
  setTimeout(sendPageView, delay)
```